Libplanet.Tests/Net/SwarmTest.Preload.cs

```diff
@@ -515,244 +527,14 @@ public async Task PreloadRetryWithNextPeers(int blockCount)
-            finally
-            {
-                await StopAsync(minerSwarm);
-                await StopAsync(receiverSwarm);
-            }
-        }
-
-        [Fact(Timeout = Timeout)]
-        public async Task PreloadWithBranchesAndTrustedPeers()
-        {
-            // Two miners; one trusts other one.  Test if it downloads the minimum range of blocks
-            // (instead of the entire chain from the genesis to the tip) when there are branches.
-            // See also: https://github.com/planetarium/libplanet/issues/465#issuecomment-525682219
-            Swarm<DumbAction> senderSwarm = CreateSwarm();
-            Swarm<DumbAction> receiverSwarm = CreateSwarm();
-
-            BlockChain<DumbAction> senderChain = senderSwarm.BlockChain;
-            BlockChain<DumbAction> receiverChain = receiverSwarm.BlockChain;
-
-            Block<DumbAction> g = senderChain.Genesis,
-                bp = TestUtils.MineNext(g, difficulty: 1024),
-                b2send = TestUtils.MineNext(bp, difficulty: 1024),
-                b2recv = TestUtils.MineNext(bp, difficulty: 1024),
-                b3 = TestUtils.MineNext(b2send, difficulty: 1024);
-
-            Assert.Equal(senderChain.Genesis, receiverChain.Genesis);
-
-            senderChain.Append(bp);
-            senderChain.Append(b2send);
-            senderChain.Append(b3);
-
-            _logger.Verbose("Sender chain:");
-            foreach (Block<DumbAction> block in senderChain.IterateBlocks())
-            {
-                _logger.Verbose(
-                    "#{BlockIndex} {BlockHash} (<- {PreviousHash})",
-                    block.Index,
-                    block.Hash,
-                    block.PreviousHash
-                );
-            }
-
-            receiverChain.Append(bp);
-            receiverChain.Append(b2recv);
-
-            _logger.Verbose("Receiver chain:");
-            foreach (Block<DumbAction> block in receiverChain.IterateBlocks())
-            {
-                _logger.Verbose(
-                    "#{BlockIndex} {BlockHash} (<- {PreviousHash})",
-                    block.Index,
-                    block.Hash,
-                    block.PreviousHash
-                );
-            }
-
-            try
-            {
-                await StartAsync(senderSwarm);
-                await receiverSwarm.AddPeersAsync(new[] { senderSwarm.AsPeer }, null);
-                await receiverSwarm.PreloadAsync(
-                    trustedStateValidators: ImmutableHashSet<Address>.Empty.Add(senderSwarm.Address)
-                );
-            }
-            finally
-            {
-                await StopAsync(senderSwarm);
-            }
-
-            Assert.Equal(senderChain.BlockHashes, receiverChain.BlockHashes);
-            // FIXME: these should be tested via storage
-            /*
-            Assert.DoesNotContain(g.Hash, receivedBlockStates)
-            Assert.DoesNotContain(bp.Hash, receivedBlockStates)
-            */
-        }
-
```

Comment:
I removed this tests because it became unable to receive states from trusted peers.